### ISSUE TITLE
Installed 0.107.7 and tried to add new Wyze Sense and it fails

### ISSUE THREAD

**Issue by DesertDwellerEv** [kevinvincent/ha-wyzesense#120]
Wyze Scan reports:
wyzesense
Sensor found and added as: binary_sensor.wyzesense_ (unless you have customized the entitiy id prior).
To add more sensors, call wyzesense.scan again.

More Info: type=1, version=16

There is no entity with that matching name in config yaml or custom yaml

**Comment by DesertDwellerEv** [comment]
Looking at logs:
Log Details (ERROR)
Logger: custom_components.wyzesense.wyzesense_custom
Source: custom_components/wyzesense/wyzesense_custom.py:364
First occurred: 7:42:54 PM (2 occurrences)
Last logged: 7:42:54 PM

Invalid packet: b'55aa531d00000001714311b7ff0000000000000000000119500101000001430555'
Mismatched checksum, remote=0555, local=049B

**Comment by VarenDerpsAround** [comment]
https://github.com/kevinvincent/ha-wyzesense/issues/121#issuecomment-610958312

**Comment by VarenDerpsAround** [comment]
essentially, you have a sensor that is out of range or is not working correctly. Follow the usual steps to remove any sensors that no longer function. You can check this with the rssi in the entities list.

**Comment by DesertDwellerEv** [comment]
Mmm, that is what I was trying to do before 0.107.7
One of the sensors was acting wonky so I deleted and updated to the latest.
So I tried a different sensor with the same result.
I am going to nuke everything for Wyze, reinstall and try again.

**Comment by ds141414** [comment]
Hello everyone, I have been struggling with this for a few days and at some stage I was disappointed it did not work.  Then I found this: 

This worked for me. Steps below:
1. Plug in the bridge and wait until the bridge light is solid blue.
2. Hold down the button with the WYZE logo on the back of the bridge until the light turns solid yellow, then let go of the button.
3. In the app, tap the 3 dots to add a product, select your device and the camera. It will say, Wyze bridge is ready to be connected and flash yellow and white.
4. Use the pin to hold the reset button in on your sensor until it flashes 3 times (my sensors flashed red 5 times). The connection was successful. Repeat steps 2-4 for other devices.

It came from one of the Wyze forms
https://forums.wyzecam.com/t/new-sense-kit-will-not-connect/28130/16

At step 3 what I did was to restart homeassistant and make sure the configuration has the correct hidraw device .  You can find that out from logging in Homeassistant with putty and enter this **command:  dmesg | grep hidraw**

Hopefully this helps someone.  

How have to figure out where to put the sensors and most likely buy more.

**Comment by DesertDwellerEv** [comment]
Gave up and removed/installed ha-wyzesense and associated json file.
Restarted and tried to sync the same Wyze Sense.
Same result. Thought I'd have to remove/add each one, but it turns out all I had to do was tickle each sensor, open/close door or walk past pir, and they were automatically added.
After failed adding 2 sensor, the 3rd, 4th, and another PIR were added without a hitch.
Calling it done and will just not use the first two sensors.
Thanks for the feedback.
